sdk-extension/opentelemetry-sdk-extension-aws/src/opentelemetry/sdk/extension/aws/trace/propagation/aws_xray_format.py

```diff
@@ -267,7 +267,7 @@ def _parse_sampled_flag(sampled_flag_str):
 
     def inject(
         self,
-        set_in_carrier: Setter[TextMapPropagatorT],
+        setter: Setter[TextMapPropagatorT],
```

Comment:
The setter here also appears to be in the wrong order

Reply:
fixed